### PR TITLE
PHrytsyshyn page object + location

### PR DESCRIPTION
**Pull request by hrytsyshynq**

### CODE REVIEW

File: src/test/java/selenium/page/CandidatesPage.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        return new CreateCandidatePage();
+    }
+
+    public CandidatesPage checkCandidate (String candidateName, String vacancyName) {
+        isCreated = usersList.stream()
+                .map(WebElement::getText)
+                .anyMatch(item -> item.contains(candidateName) &&
+                        item.contains(vacancyName));
+        return this;
+    }
```
Comment (alexyablonskyi) [inline src/test/java/selenium/page/CandidatesPage.java:28]: Method that verify something should return boolean, not page object.

File: src/test/java/selenium/test/CandidatesPageTest.java [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+        CandidatesPage page = getLoginPage()
+                .fillUsername(getCredentials().getUserLoginName())
+                .fillPassword(getCredentials().getUserPassword())
+                .clickLoginButton()
+                .moveToRecruitmentItem()
+                .clickCandidateListButton()
+                .checkCandidate(getUser().getUserLastName(), getVacancy().getVacancyName());
+
+        boolean isCreated = page.isCreated();
+        Assert.assertTrue(isCreated);
```
Comment (alexyablonskyi) [inline src/test/java/selenium/test/CandidatesPageTest.java]: Assertion message is missing

File: src/test/java/selenium/test/CreateCandidatePageTest.java [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+                .fillDate(candidate.getCandidateAppliedDate())
+                .btnSaveClick();
+
+
+        if (page.checkValidation()) {
+            String successMessage = page.getSuccessMessage();
+            Assert.assertEquals(successMessage, "Successfully Saved\nClose");
+        } else {
+            Assert.fail("Check required fields! Fields message: \n" + page.getValidationMessage());
+        }
```
Comment (alexyablonskyi) [inline src/test/java/selenium/test/CreateCandidatePageTest.java]: Please avoid if/else in assertion part, expected result should be only one

File: src/test/java/selenium/test/CreateLocationPageTest.java [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+                .fillLocationName(getLocation().getName())
+                .fillCountry(getLocation().getCountry())
+                 .fillCity(getLocation().getCity())
+                .fillPhone(getLocation().getPhone())
+                .saveLocation();
+
+        boolean isCreated = page.toLocationPage()
+                .checkLocation(getLocation().getName(), getLocation().getCity());
+        Assert.assertTrue(isCreated);
+    }
```
Comment (alexyablonskyi) [inline src/test/java/selenium/test/CreateLocationPageTest.java:27]: Missed assertion message. Please add it in each test class.

File: src/test/java/selenium/test/CreateVacancyPageTest.java [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+                .setJobStatus()
+                .setPublishedInFeedStatus()
+                .saveVacancy();
+
+        if (page.checkValidation()) {
+            String successMessage = page.getSuccessMessage();
+            Assert.assertEquals(successMessage, "Successfully Saved\nClose");
+        } else {
+            Assert.fail("Check required fields! Fields message: \n" + page.getValidationMessage());
+        }
```
Comment (alexyablonskyi) [inline src/test/java/selenium/test/CreateVacancyPageTest.java]: Here is the same remark as comment above.

**Review by alexyablonskyi — COMMENTED** [review]
Fix remarks and you can merge pull request.

File: src/test/java/selenium/page/CandidatesPage.java (hunk 1 above)
Comment (hrytsyshynq) [reply src/test/java/selenium/page/CandidatesPage.java:28]: Fixed

File: src/test/java/selenium/test/CandidatesPageTest.java (hunk 2 above)
Comment (hrytsyshynq) [reply src/test/java/selenium/test/CandidatesPageTest.java]: Fixed

File: src/test/java/selenium/test/CreateCandidatePageTest.java (hunk 3 above)
Comment (hrytsyshynq) [reply src/test/java/selenium/test/CreateCandidatePageTest.java]: Fixed

File: src/test/java/selenium/test/CreateLocationPageTest.java (hunk 4 above)
Comment (hrytsyshynq) [reply src/test/java/selenium/test/CreateLocationPageTest.java:27]: Fixed

File: src/test/java/selenium/test/CreateVacancyPageTest.java (hunk 5 above)
Comment (hrytsyshynq) [reply src/test/java/selenium/test/CreateVacancyPageTest.java]: Fixed

**Pull request merged**